src/index.html

```diff
@@ -11,8 +11,54 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The `href` attribute is pointing to a SCSS file (`index.scss`). Browsers cannot directly interpret SCSS files. You should compile the SCSS to CSS and link to the resulting CSS file instead.